sql/verifier.go

```diff
@@ -7,7 +7,7 @@ import (
 
 // fieldTypes[field][table]type.  For more information, please check
 // verifier_test.go.
-type fieldTypes map[string]map[string]string
+type fieldTypes map[string]map[string](*columnType)
```

Comment:
why do we need this change? Looks like `columnType.Name` is redundant.

Reply:
You're right.
It is an outdated definition.